### ISSUE TITLE
handling of disappearing and reappearing device

### ISSUE THREAD

**Issue by christf** [livkx/cadmus#13]
when cadmus has to deal with inputs that get removed and later re-added, it would be nice if the denoising would seamlessly resume instead of breaking the filter pipeline.

**Comment by livkx** [comment]
Hi! Please can you further explain this issue, not sure I fully understand. Are you asking that Cadmus watches and waits for the selected input to re-appear so that you can keep it's input selected as the microphone in an app, and it will seamlessly use the original selected input device once it becomes connected again?

**Comment by christf** [comment]
indeed. This seems to be what browsers are doing to organize their mic inputs for video conferencing. I only noticed because this does not seem to happen when cadmus is involved...
